vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -48,18 +66,32 @@
     become_user: root
 
   - name: Wait until rebooted worker "{{ WORKER_NODE }}" becomes NotReady
```

Comment:
I think it will always work, because when we reboot a BMH, corresponding libvirt VM always gets power cycled. And when libvirt VM is off, k8s node becomes unresponsive which is detected fast enough. However, I assume that if livbirt VM turns on very fast that k8s doesn't detect it, then we might get stuck and fail eventually here.

Reply:
Looks like we actually had [this kind of error](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_ubuntu/559/console) in the CI today. :slightly_frowning_face:
Do you think it would improve if we set `delay` to some smaller number? The libvirt task above has `delay: 10` which I think is quite high so it could quite easily miss it. Maybe 3 or even 1 would work better?

Reply:
Sounds good to me